py/dml/toplevel.py

```diff
@@ -70,7 +70,7 @@ def determine_version(filestr, filename):
         column = ver_start - filestr.rfind('\n', 0, ver_start)
         m = check_version.match(filestr, pos=ver_start)
         if not m:
-            raise ESYNTAX(SimpleSite(filename, lineno, column),
+            raise ESYNTAX(SimpleSite("%s:%d:%d" % (filename, lineno, column)),
```

Comment:
You're being inconsistent about the use of f-strings vs `%` when creating `SimpleSite`s, I'm ok with either alternative, but alternating between both in the same PR for the same use-case is very strange to me.

Reply:
This comes from a commit that is a pure revert, I didn't bother to preserve the f-string migration parts of the reverted commit.

Reply:
(in general, f-string should be the default choice that should be used unless there's a good reason why % is better.)